src/main/java/com/vinylteam/vinyl/entity/Shop.java

```diff
@@ -0,0 +1,69 @@
+package com.vinylteam.vinyl.entity;
+
+import java.util.Objects;
+
+public class Shop {
+
+    private int id;
+    private String mainPageLink;
+    private String imageLink;
```

Comment:
I suggest to rename to logoLink

Reply:
It is called link_to_image in db. Gotta change it too to match - so maybe later.